### DISCUSSION TITLE
Unable to locate module 'swiper/css' or corresponding type declarations Version 9.0.5

### DISCUSSION THREAD

**Discussion by hakimLyon** [nolimits4web/swiper#6421] [Q&A] [answered]
How to import swiper css ?

![Capture d’écran du 2023-02-21 14-24-32](https://user-images.githubusercontent.com/64654633/220372241-854d4a29-4849-46ee-9115-56471c78b27e.png)

**Comment by hajirasyafi** [comment, marked as answer]
like this :
`import 'swiper/css';`

if you want to use navigation and pagination you have to import it :
`import 'swiper/css/navigation';`
`import 'swiper/css/pagination';`

see : https://swiperjs.com/react

**Reply by hakimLyon** [reply]
thank you , this work for me ``` import swipperCss from  'swiper/swiper-bundle-min.css' ```

**Reply by hajirasyafi** [reply]
oh great, but if you import that bundle that mean you import all css modules, for best performance just import necessary css modules.

**Reply by hakimLyon** [reply]
oh...  you're right, thanks 😁.
